packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -123,7 +176,27 @@ export class TestRunnerService {
 				continue;
 			}
 
-			// TODO: 2.3 Collect the run data
+			// Collect the results of the test case execution
+			const testCaseRunData = execution.data.resultData.runData;
```

Comment:
Renaming `execution` into `testCaseExecution` could distinguish it from `executionData` bit more